theano/printing.py

```diff
@@ -642,11 +642,20 @@ def pydotprint(fct, outfile=None,
     label each edge between an input and the Apply node with the
     input's index.
 
-    Green boxes are inputs variables to the graph,
-    blue boxes are outputs variables of the graph,
-    grey boxes are variables that are not outputs and are not used,
-    red ellipses are transfers from/to the gpu (ops with names GpuFromHost,
-    HostFromGpu).
+    Variable color code::
```

Comment:
I think this is the syntax for introducing a code-block. If I remember correctly, the "-" below should be aligned just below "Variable", and there should be only one ":" here.